docs/getting-started/installation.rst

```diff
@@ -46,20 +46,27 @@ Installing the "bleeding edge" version from GitHub
 --------------------------------------------------
 
 You can install the very latest, "bleeding edge", development version of
-PyMedPhys by directly cloning its GitHub repo and installing its dependencies
-using the following commands from within a terminal:
+PyMedPhys, but you will first need Git and Yarn. In turn, Yarn requires Nodejs.
+Perhaps the easiest way to install these is to install Chocolatey from within
+a terminal in Administrator mode using `these instructions <https://chocolatey.org/docs/installation>`__.
```

Comment:
Actually, maybe we should keep this one OS independent. If someone is wanting to install bleeding edge they can work some things out.

Potentially the following link should be sufficient:

https://yarnpkg.com/en/docs/install

Reply:
Good point!